### PR TITLE
systemd: additional hardening of service files

### PR DESCRIPTION
**Pull request by smithfarm**
As the systemd developers introduce security features, we endeavor to put them
to use for the benefit of security-thirsty users.

Signed-off-by: Nathan Cutler <ncutler@suse.com>

### CODE REVIEW

File: usr/lib/systemd/system/rbd-target-gw.service [hunk 1]
```diff
@@ -19,6 +19,12 @@ PrivateDevices=yes
 ProtectHome=true
 ProtectSystem=full
 PrivateTmp=true
+ProtectHostname=true
+ProtectClock=true
+ProtectKernelTunables=true
```
[the remark this answers is not in the document]
Comment (jsegitz) [reply usr/lib/systemd/system/rbd-target-gw.service:24]: it's possible to whitelist individual writeable paths, but in this case I would just leave this out for now

**Comment by jsegitz** [comment]
no testing from my side either, I do this mass scale for openSUSE and rely on the packagers for testing.

**Comment by smithfarm** [comment]
UPDATE: dropped `ProtectKernelTunables=true`

**Comment by lxbsz** [comment]
Run some tests without using the containers, the basic functions worked well for me, but the `gwcli ls` will stuck for a long time and I can reproduce this very easy.

**Pull request closed without merging**